index.bs

```diff
@@ -1499,7 +1499,7 @@ a variable <var>stream</var>, that performs the following steps:
     </ol>
   <li> Set <var>stream</var>@\[[writing]] to <b>true</b>.
   <li> Let <var>writeResult</var> be PromiseInvokeOrNoop(<var>stream</var>@\[[underlyingSink]], <code>"write"</code>,
-    «‍<var>writeRecord</var>.\[[chunk]]»).
+    « <var>writeRecord</var>.\[[chunk]]»).
```

Comment:
Oh weird I was using some multibyte left angle quote here... you inserted a space but you also fixed that. So strange.